dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HadoopUtils.java

```diff
@@ -612,7 +607,9 @@ public static String getAppAddress(String appAddress, String rmHa) {
             return null;
         }
 
-        String end = Constants.COLON + split2[1];
+        String activeResourceManagerPort = String.valueOf(PropertyUtils.getInt(Constants.HADOOP_RESOURCE_MANAGER_HTTPADDRESS_PORT, 8088));
+        String rest = split2[1].substring(activeResourceManagerPort.length());
+        String end = Constants.COLON + activeResourceManagerPort + rest;
```

Comment:
1. How do you configure this property 'yarn.application.status.address' in the common.properties file?

2. In the common.properties file, the property 'yarn.application.status.address=http://ds1:8088/ws/v1/cluster/apps/%s' have configured in the port, the changes of the code is also increase the port handling, This will send a conflict.

Reply:
1 # if resourcemanager HA enable or not use resourcemanager, please keep the default value; If resourcemanager is single, you only need to replace ds1 to actual resourcemanager hostname.
yarn.application.status.address=http://ds1:8088/ws/v1/cluster/apps/%s
# if custom you  resourcemanager port ,you need to replace 8088 else default value.
resource.manager.httpaddress.port=8088
2 I think this port should be set separately

Reply:
I understand what you think. At present, there are two attributes that represent yarn resource manager HTTP port. It is suggested to change them to one

Reply:
Right, don’t you know it’s solved now?

Reply:
In  the common.properties file there are two properties(yarn.application.status.address=http://ds1:8088/ws/v1/cluster/apps/%s;resource.manager.httpaddress.port=8088) that both represent the yarn resource manager http port. It is recommended to use only one property to represent.


Solution 1: Keep the status quo. When customizing the port, you need to modify 'resource.manager.httpaddress.port=8088' and 'yarn.application.status.address=http://ds1:8088/ws/v1/cluster/apps /%s' port value
Solution 2:'yarn.application.status.address=http://ds1:8088/ws/v1/cluster/apps/%s' is modified to 'yarn.application.status.address=http://ds1:%s /ws/v1/cluster/apps/%s', use 'resource.manager.httpaddress.port=8088' in the code to replace the port value

Reply:
I agree Solution 2,I will revise and submit again